<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbMode">
     <property name="title">
      <string>Modus</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbSuperMelee">
        <property name="text">
         <string>Supermelée</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbTeams">
        <property name="text">
         <string>Feste Teams</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbTete">
        <property name="text">
         <string>Tête</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>210</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPoints">
     <property name="title">
      <string>Punkte</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="rbFormuleX">
        <property name="text">
         <string>Formule X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSwissSimple">
        <property name="text">
         <string>Vereinf. Schweizer System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSwissBuchholz">
        <property name="text">
         <string>Schweizer System mit Bucholz-Wertung</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>12</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="tlDataDir">
       <property name="text">
        <string>Daten-Verzeichnis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leDataDir</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDataDir"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbSelectDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="tlFontSize">
       <property name="text">
        <string>Font-Größe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFontSize"/>
     </item>
     <item>
      <widget class="QLabel" name="tlPoint">
       <property name="text">
        <string>pt</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="cbSiteEnabled">
       <property name="text">
        <string>Plätze berücksichtigen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="tlSiteCount">
       <property name="text">
        <string>Anzahl Plätze</string>
       </property>
       <property name="buddy">
        <cstring>leSiteCount</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSiteCount"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbTeamsOnly">
     <property name="text">
      <string>Runde nur mit Teams anzeigen (ohne Spieler)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="tlPassword">
       <property name="text">
        <string>Passwort</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lePassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tlPasswordCopy">
       <property name="text">
        <string>Wiederholung</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lePasswordCopy">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbSuperMelee</tabstop>
  <tabstop>rbTeams</tabstop>
  <tabstop>rbFormuleX</tabstop>
  <tabstop>rbSwissSimple</tabstop>
  <tabstop>rbSwissBuchholz</tabstop>
  <tabstop>leDataDir</tabstop>
  <tabstop>tbSelectDir</tabstop>
  <tabstop>leFontSize</tabstop>
  <tabstop>cbSiteEnabled</tabstop>
  <tabstop>leSiteCount</tabstop>
  <tabstop>cbTeamsOnly</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>lePasswordCopy</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
